Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -50,37 +142,72 @@ public class Custom2DPuckExample: UIViewController, ExampleProtocol {
         finish()
     }
 
-    @objc func showHideAccuracyRadius() {
-        showsAccuracyRing.toggle()
+    private func customizePuckButton() {
+        // Set up button to change the puck options
+        let button = UIButton(type: .system)
+        button.setTitle("Customize puck", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.backgroundColor = #colorLiteral(red: 0, green: 0.4784313725, blue: 0.9882352941, alpha: 1)
+        button.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
It is a good practice to keep `translatesAutoresizingMaskIntoConstraints = false` close to the constraints setup. It would help you not to forget this part on next views.